module.txt

```diff
@@ -1,9 +1,9 @@
 {
     "id" : "ItemRendering",
     "version" : "1.0.0-SNAPSHOT",
-    "author" : "Josharias",
+    "author" : "Josharias, msteiger, Cervator, skaldarnar",
     "displayName" : "ItemRendering",
-    "description" : "",
+    "description" : "A library for displaying "holographic" items (i.e. spinning items when dropped on the floor) in Terasology",
```

Comment:
1. "[holographic](https://en.wikipedia.org/wiki/Holography)" does not mean "spinning when dropped on the floor", so this sentence doesn't make too much sense.

2. there are other effects in the module (bounce, wobble etc.)